skopt/learning/forest.py

```diff
@@ -194,7 +194,7 @@ class RandomForestRegressor(_sk_RandomForestRegressor):
     .. [1] L. Breiman, "Random Forests", Machine Learning, 45(1), 5-32, 2001.
 
     """
-    def __init__(self, n_estimators=10, criterion='mse', max_depth=None,
+    def __init__(self, n_estimators=10, criterion='squared_error', max_depth=None,
                  min_samples_split=2, min_samples_leaf=1,
```

Comment:
To respect the 80 line width restriction, I propose to put `max_depth` on the next line.

```suggestion
    def __init__(self, n_estimators=10, criterion='squared_error',
                 max_depth=None, min_samples_split=2, min_samples_leaf=1,
```

Reply:
Looks good